ppod.py

```diff
@@ -30,8 +35,12 @@ def lambda_handler(event: dict, context: object) -> dict:
             xml_files = extract_files_from_tar(s3_file_content)
             for xml_file in xml_files:
                 if xml_file:
-                    add_namespaces_to_alma_marcxml(xml_file)
-                    # post modified_xml to POD
+                    modified_xml = add_namespaces_to_alma_marcxml(xml_file)
+                    pod_file_name = os.path.basename(s3_file).replace("tar.gz", "xml")
+                    response = post_files_to_pod(
+                        url, headers, pod_file_name, modified_xml
+                    )
+                    logger.info("%s: %s", pod_file_name, response)
```

Comment:
Could we say what happened in this log message? Something like file X uploaded with response Y?